gui/services/models.py

```diff
@@ -1164,6 +1164,10 @@ class SNMP(Model):
         blank=True,
         help_text=_("These parameters will be added to /etc/snmpd.config."),
     )
+    snmp_zilstat = models.BooleanField(
+        verbose_name=_("Expose zilstat via SNMP"),
+        default=False,
```

Comment:
We should have a help text warning about performance implications of having this enabled.